aion/web/controllers/question_transactions.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Aion.QuestionTransactions do
+  @moduledoc """
+  Implements transactions used by Aion.QuestionController
+  """
+  use Aion.Web, :controller
+
+  alias Aion.Question
+  alias Aion.Answer
+  alias Aion.Subject
+  alias Ecto.Multi
+
+  @answer_separator ","
+
+  def create_question_with_answers(question, answers, subject_id) do
+      Multi.new
+      |> Multi.run(:create_changeset, fn _ -> create_question_changeset(subject_id, question) end)
```

Comment:
How about using Ecto.Multi.insert here? Also I don't think this lambda is needed here, you just need to pass a changeset (I mean you could just call create_question_changeset here).